tests/nlu/base/test_evaluation.py

```diff
@@ -320,6 +326,59 @@ def test_intent_evaluation_report(tmpdir_factory):
     assert result["predictions"][0] == prediction
 
 
+def test_intent_evaluation_report_large(tmpdir_factory):
+    path = tmpdir_factory.mktemp("evaluation").strpath
+    report_folder = os.path.join(path, "reports")
+    report_filename = os.path.join(report_folder, "intent_report.json")
+
+    rasa.utils.io.create_directory(report_folder)
```

Comment:
or make `create_directory` compatible with `Path` 
```suggestion
    rasa.utils.io.create_directory(str(report_folder))
```